src/britive/notification_mediums.py

```diff
@@ -0,0 +1,65 @@
+class NotificationMediums:
+    def __init__(self, britive):
+        self.britive = britive
+        self.base_url = f'{self.britive.base_url}/v1/notification-service/notificationmediums'
+    def list(self):
+        """
+        Provide a list of all notification mediums 
+        :return: List of all notification mediums
+        """
+        return self.britive.get(self.base_url)
+
+    def create(self, type : str, name : str, description : str = "Default notification medium description", connectionParameters : dict = {}) -> dict:
+        """
+        Create a new notification medium.
+        :param 
+            type : the type of the notification medium
+            name : the name of the notification medium
+            description : the description of the notification medium
+            connectionParameters : the connection parameters of the notification medium
+                valid connection parameters: 
+                    URL : for slack, the URL
+                    token : for slack, Auth Token for the Application BOT
+                    Webhook URL : for teams, the URL of the teams webhook
+        :return: Details of the newly created notification medium.
+        """
+        params = {'name': name, 'description': description, 'type': type, 'connectionParameters': connectionParameters}
```

Comment:
make multi line

Reply:
fixed in 637defb2698c5507cf1d0bc494707e7badd46d13